docs/about_Puppet.Dsc.md

```diff
@@ -0,0 +1,70 @@
+# Puppet.Dsc
+## About Puppet.Dsc
+
+# SHORT DESCRIPTION
+
+This module provides a means of converting a PowerShell module containing DSC resources into a Puppet module which vendors those DSC resources as Puppet types and providers.
+
+# LONG DESCRIPTION
+
+This module provides a means of converting a PowerShell module containing DSC resources into a Puppet module which vendors those DSC resources as Puppet types and providers.
+
+In rough order it:
+
+1. Scaffolds out a new Puppet module
+1. Downloads a copy of the specified PowerShell module and vendors it (and all of its dependencies) into the Puppet module
+1. Updates the Puppet module based on the PowerShell module's metadata
+1. Converts the DSC resources into Puppet Resource API types
+1. Writes the appropriate types and providers into the Puppet module
```

Comment:
Are these lines the same?

Maybe the second line is `Generates...`?

Reply:
I don't _think_ so, because parsing and translating the properties as needed is different from writing files? 🤔